packages/expo/package.json

```diff
@@ -44,7 +44,9 @@
     "typescript": "*"
   },
   "peerDependencies": {
-    "react": ">=16"
+    "react": ">=16",
+    "expo-auth-session": ">=4",
```

Comment:
Did we always implicitly require these versions? 
Just checking as otherwise `npm i` might throw errors about conflicting dependency versions.

Reply:
The expo-* dependencies are new.